devfiles/java-jboss-eap-xp-bootable-jar/devfile.yaml

```diff
@@ -0,0 +1,173 @@
+starterProjects:
+  - name: microprofile-config
+    description: EAP XP 2.0 Eclipse Microprofile Config Quickstart
+    git:
+      checkoutFrom:
+        remote: jboss-developer
+        revision: xp-2.0.x
+      remotes:
+        jboss-developer: https://github.com/jboss-developer/jboss-eap-quickstarts.git
+    subDir: microprofile-config
+  - name: microprofile-fault-tolerance
+    description: EAP XP 2.0 Eclipse Microprofile Fault Tolerance Quickstart
+    git:
+      checkoutFrom:
+        remote: jboss-developer
+        revision: xp-2.0.x
+      remotes:
+        jboss-developer: https://github.com/jboss-developer/jboss-eap-quickstarts.git
+    subDir: microprofile-fault-tolerance
+  - name: microprofile-health
+    description: EAP XP 2.0 Eclipse Microprofile Health Quickstart
+    git:
+      checkoutFrom:
+        remote: jboss-developer
+        revision: xp-2.0.x
+      remotes:
+        jboss-developer: https://github.com/jboss-developer/jboss-eap-quickstarts.git
+    subDir: microprofile-health
+  - name: microprofile-jwt
+    description: EAP XP 2.0 Eclipse Microprofile JWT Quickstart
+    git:
+      checkoutFrom:
+        remote: jboss-developer
+        revision: xp-2.0.x
+      remotes:
+        jboss-developer: https://github.com/jboss-developer/jboss-eap-quickstarts.git
+    subDir: microprofile-jwt
+  - name: microprofile-metrics
+    description: EAP XP 2.0 Eclipse Microprofile Metrics Quickstart
+    git:
+      checkoutFrom:
+        remote: jboss-developer
+        revision: xp-2.0.x
+      remotes:
+        jboss-developer: https://github.com/jboss-developer/jboss-eap-quickstarts.git
+    subDir: microprofile-metrics
+  - name: microprofile-openapi
+    description: EAP XP 2.0 Eclipse Microprofile OpenAPI Quickstart
+    git:
+      checkoutFrom:
+        remote: jboss-developer
+        revision: xp-2.0.x
+      remotes:
+        jboss-developer: https://github.com/jboss-developer/jboss-eap-quickstarts.git
+    subDir: microprofile-openapi
+  - name: microprofile-opentracing
+    description: EAP XP 2.0 Eclipse Microprofile OpenTracing Quickstart
+    git:
+      checkoutFrom:
+        remote: jboss-developer
+        revision: xp-2.0.x
+      remotes:
+        jboss-developer: https://github.com/jboss-developer/jboss-eap-quickstarts.git
+    subDir: microprofile-opentracing
+  - name: microprofile-rest-client
+    description: EAP XP 2.0 Eclipse Microprofile REST Client Quickstart
+    git:
+      checkoutFrom:
+        remote: jboss-developer
+        revision: xp-2.0.x
+      remotes:
+        jboss-developer: https://github.com/jboss-developer/jboss-eap-quickstarts.git
+    subDir: microprofile-rest-client
+components:
+  - name: jaeger
+    container:
+      image: quay.io/jaegertracing/all-in-one:1.21.0
+      memoryLimit: 128Mi
+      endpoints:
+        - name: 'tracing-ui'
+          targetPort: 16686
+  - name: eap-xp
+    container:
+      image: registry.redhat.io/ubi8/openjdk-11
```

Comment:
This might be a problem :-(
Devfiles in default odo registry (this registry) should work on vanilla Kubernets. This image is sadly not accessible without Red Hat loggin :-(

Reply:
This could be replaced by `registry.access.redhat.com/ubi8/openjdk-11` that does not require authentication

Reply:
ubi should be available without authentication
https://catalog.redhat.com/software/containers/ubi8/openjdk-11/5dd6a4b45a13461646f677f4?container-tabs=gti&gti-tabs=unauthenticated

Reply:
> ubi should be available without authentication
> https://catalog.redhat.com/software/containers/ubi8/openjdk-11/5dd6a4b45a13461646f677f4?container-tabs=gti&gti-tabs=unauthenticated

Yes, but with one difference. It needs to be downloaded as `registry.access.redhat.com/ubi8/openjdk-11`
Using it from `registry.redhat.io/ubi8/openjdk-11` will still require login.